packages/apps-routing/src/index.ts

```diff
@@ -70,7 +71,7 @@ const routes: Routes = appSettings.uiMode === 'light'
     sudo,
     null,
     settings,
-    toolbox,
+    // toolbox,
```

Comment:
What about all the commented lines in this file? Would you remove them, please? If they are necessary later for a feature can you use babel conditional-compile and disable them using a feature flag?

Reply:
I will remove these for now.. we can add them again from the history..